pkg/constants/constants.go

```diff
@@ -134,6 +134,47 @@ const (
 
 	// TorchEnvMasterPort is the env name for the master node port.
 	TorchEnvMasterPort string = "PET_MASTER_PORT"
+
+	// TochTuneArgNumNodes is the arg anme for the number of training nodes.
+	TorchTuneArgNumNodes string = "--nnodes"
+
+	// TorchTuneArgNumProcPerNode is the arg name for the number of procs per node (e.g. number of GPUs per Pod).
+	TorchTuneArgNumProcPerNode string = "--nproc_per_node"
+
+	// TorchTuneArgRdzvId is the arg name for the rendezvous ID.
+	TorchTuneArgRdzvId string = "--rdzv_id"
+
+	// TorchTuneArgRdzvEndpoint is the arg name for the rendezvous endpoint.
+	TorchTuneArgRdzvEndpoint string = "--rdzv_endpoint"
+
+	// TorchTuneFullFinetuneSingleDevice Recipe is the recipe for the single device full finetune.
+	TorchTuneFullFinetuneSingleDevice string = "full_finetune_single_device"
+
+	// TorchTuneFullFinetuneSingleDeviceConfigSuffix is the config suffix for the single device full finetune.
+	TorchTuneFullFinetuneSingleDeviceConfigSuffix string = "_full_single_device"
+
+	// TorchTuneFullFinetuneDistributed Recipe is the recipe for the distributed full finetune.
+	TorchTuneFullFinetuneDistributed string = "full_finetune_distributed"
+
+	// TorchTuneFullFinetuneMultiDevicesConfigSuffix is the config suffix for the single node distributed full finetune.
+	TorchTuneFullFinetuneMultiDevicesConfigSuffix string = "_full"
+
+	// TorchTuneFullFinetuneMultiNodesConfigSuffix is the config suffix for the multi node distributed full finetune.
+	TorchTuneFullFinetuneMultiNodesConfigSuffix string = "_full_multinode"
+
+	// TorchTuneDefaultRecipe is the default recipe for the torchtune.
+	TorchTuneDefaultRecipe string = TorchTuneFullFinetuneDistributed
+)
+
+const (
+	// MODEL_LLAMA3_2_1B is the model name for the Llama3.2 1B Instruct model.
+	MODEL_LLAMA3_2_1B = "llama3_2/1B"
```

Comment:
Can you keep these constants under other TorchTune const:
```suggestion
        TorchTuneModelLlama3_2_1B = "llama3_2/1B"
```

Reply:
SGTM